neo/Consensus/ConsensusService.cs

```diff
@@ -116,6 +116,7 @@ private void CheckExpectedView(byte viewNumber)
                 if (message is null || message.NewViewNumber < viewNumber)
                     localNode.Tell(new LocalNode.SendDirectly { Inventory = context.MakeChangeView(viewNumber) });
                 InitializeConsensus(viewNumber);
+                context.Save();
```

Comment:
Let's remove these improvements first, fix the bug first, and then test. Improve slowly later.

Reply:
The bug you were referring to was fixed in #641. These improvements are necessary to fix a stall that will happen too frequently without these changes.